composer.json

```diff
@@ -11,23 +11,23 @@
     ],
     "require": {
         "php": ">=5.5",
-        "aura/router": "^2.3",
         "container-interop/container-interop": "^1.1",
+        "filp/whoops": "^1.1",
```

Comment:
I suggest moving this out for in a production environment we don't need this beautiful stuff ;) .